packages/hoc-withtooltip/README.md

```diff
@@ -0,0 +1,31 @@
+# Farmblocks HOC-withTooltip
+
+A Higher Order Component that handles tooltip visibility.
+
+## Installation
+
+```
+npm install @crave/farmblocks-hoc-withtooltip
+```
+
+## Usage
+
+```
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+import withTooltip from '@crave/farmblocks-hoc-withTooltip'
+
+const InputWithTooltip = withTooltip("input");
+
+const App = props => (
+  <div>
+    <InputWithTooltip tooltipText="This is the text rendered inside the tooltip" />
```

Comment:
`tooltipText` has changed to `tooltipContent `.

Readme should point out that this hoc uses the tooltip component and which components we are expecting